sky/engine/core/rendering/style/RenderStyleConstants.h

```diff
@@ -154,6 +154,10 @@ enum ETextAlign {
     LEFT, RIGHT, CENTER, JUSTIFY, TASTART, TAEND,
 };
 
+enum EFlutterLineBreakMode {
+    STANDARD, ELLIPSIZE
```

Comment:
C++ enums are dumb.  This creates a global symbol called STANDARD.  In CSS, this is called "clip"...  Dunno.